CHANGES

```diff
@@ -5,6 +5,31 @@
   or the CRS mailinglist at
 * https://lists.owasp.org/mailman/listinfo/owasp-modsecurity-core-rule-set
 
+== Version 3.2.4 - 2023-10-27 ==
+
+Important changes:
+ * Backported important Fixes from v3.3
+
+Fixes and improvements:
```

Comment:
As I see you made the fixes which described "CRS bypass" issues, which is fine for me.